pkg/solana/chain.go

```diff
@@ -288,13 +289,25 @@ func newChain(id string, cfg *config.TOMLConfig, ks loop.Keystore, lggr logger.L
 
 		// clientCache will not be used if multinode is enabled
 		ch.clientCache = nil
-	}
 
-	tc := func() (client.ReaderWriter, error) {
-		return ch.getClient()
+		// Send tx using MultiNode transaction sender
+		sendTx = func(ctx context.Context, tx *solanago.Transaction) (solanago.Signature, error) {
+			// Use empty context since individual RPC timeouts are handled by the client
+			result := ch.txSender.SendTransaction(context.Background(), tx)
```

Comment:
We should not use context.Background. 
I agree that it does not make sense to override RPC timeouts, but the fact that we can not handle this case signals a deeper problem. 
I've described what seems to be a root cause in another comment.

Reply:
Why doesn't it make sense to pass the parent context here? We can at least use `context.WithoutCancel` to maintain the request scoped data, but that must be justified, and all behavior should be cancelled on shutdown besides.
```suggestion
			result := ch.txSender.SendTransaction(context.WithoutCancel(ctx), tx)
```